assets/js/modules/heroImages.js

```diff
@@ -0,0 +1,50 @@
+/* global $ */
+'use strict';
+
+const SELECTORS = {
+    parent: '.js-random-hero',
+    image: '.js-random-hero__image',
+    caption: '.js-random-hero__caption'
+};
+
+const rand = xs => xs[Math.floor(Math.random() * xs.length)];
+
+function replaceImage($imageEl, randomImage) {
+    $imageEl.html(`
+        <picture>
+            <source srcset="${randomImage.large}" media="(min-width: 760px)">
```

Comment:
Is this `760` tied to the tablet breakpoint? It's available in JS – have a look at line 6 of `carousel.js`. Just in case we want to make this dynamic.

Reply:
I've just changed this so it's now far less tied to the general breakpoints…otherwise yes!